main.go

```diff
@@ -142,7 +150,7 @@ func main() {
 
 	switch command {
 	case stShowCmd.FullCommand():
-		href, err := paramToHref("server_templates", *stShowNameOrHref, 0)
+		href, err := paramToHref("server_templates", *stShowNameOrHref, 0, false)
```

Comment:
I think we should allow a local path for show.

Reply:
Sure, Will work on adding `local path` capability for `show`. 
Question:  Do we want to allow multiple inputs for `show` ?

Reply:
We don't have it already, so I don't think we need to add it now.